sdk/eidolon_ai_sdk/cpu/llm/open_ai_llm_unit.py

```diff
@@ -273,3 +274,48 @@ def _convert_tool_call(tool: Dict[str, any]) -> ToolCall:
     except json.JSONDecodeError as e:
         raise RuntimeError(f"Error decoding response function arguments for tool {name}") from e
     return ToolCall(tool_call_id=tool["id"], name=name, arguments=loads)
+
+
+async def _openai_completion(*args, **kwargs):
+    return await AsyncOpenAI().chat.completions.create(*args, **kwargs)
+
+
+async def _raw_parser(resp):
```

Comment:
each replay point needs to specify its own parser. By default we will just stringily the response if no parser is provided, but this gives us much better feel when using the cli.